src/Panel/EnvironmentPanel.php

```diff
@@ -48,6 +48,7 @@ protected function _prepare()
             'CACHE' => CACHE,
             'CAKE' => CAKE,
             'CAKE_CORE_INCLUDE_PATH' => CAKE_CORE_INCLUDE_PATH,
+            'CONFIG' => CONFIG,
```

Comment:
This constant is defined in the application, which is why it was an application constant.